locale/en_US/submission.po

```diff
@@ -347,6 +347,9 @@ msgstr "You are not allowed to submit to this journal because authors must be re
 msgid "submission.wizard.noSectionAllowed.description"
 msgstr "You are not allowed to submit to this journal because submissions to all sections of this journal have been deactivated or restricted. If you believe this is an error, please contact <a href='mailto:{$email}'>{$name}</a>."
 
+msgid "submission.wizard.sectionClosed.message"
+msgstr "{$contextName} is not accepting submissions to the {$section} section. If you need help recovering your submission, please contact <a href='mailto:{$email}'>{$name}</a>."
```

Comment:
Here I'd replace `'mailto:{$email}'` by `\"mailto:{$email}\"` just to keep things standard :)

Also, as this message is being shared by the API and the HTML frontend, I think it makes sense to create an extra error message for the API (to not send HTML as output), or to rephrase the text (e.g. "contact Jonas at <a>email</a>") and strip the tags.